src/garrysmod/gamemodes/zombiesurvival/gamemode/howl/argument_types/duration.lua

```diff
@@ -0,0 +1,97 @@
+-- Translations of words into seconds.
+local tokens = {
+    second = 1,
+    sec = 1,
+    minute = 60,
+    min = 60,
+    hour = 60 * 60,
+    day = 60 * 60 * 24,
+    week = 60 * 60 * 24 * 7,
+    month = 60 * 60 * 24 * 30,
+    mon = 60 * 60 * 24 * 30,
+    year = 60 * 60 * 24 * 365,
+    yr = 60 * 60 * 24 * 365,
+    permanently = 0,
+    perma = 0,
+    perm = 0,
+    pb = 0,
+    forever = 0,
+    moment = 1
+}
+
+local num_tokens = {
+    one = 1,
+    two = 2,
+    three = 3,
+    four = 4,
+    five = 5,
+    six = 6,
+    seven = 7,
+    eight = 8,
+    nine = 9,
+    ten = 10,
+    few = 5,
+    couple = 2,
+    bunch = 120,
+    lot = 1000000,
+    dozen = 12
+}
+
+Howl.Command.argument_types.duration = function(value)
```

Comment:
This argument type wrapper takes a string value like "1 week" and converts it into a duration in seconds. Again, borrowed from Flux: https://github.com/TeslaCloud/flux-ce/blob/23f2c1cc5f1960fbc5ece8fe5dfecf4edb915b35/plugins/admin/plugin/lib/sh_admin.lua#L290.